core/unit_test/TestViewMapping_a.hpp

```diff
@@ -1009,9 +1009,7 @@ void test_view_mapping() {
     ASSERT_EQ(a.use_count(), 1);
     ASSERT_EQ(b.use_count(), 0);
 
-#if !defined(KOKKOS_ENABLE_CUDA) || !defined(KOKKOS_ENABLE_CUDA_LAMBDA)
-    // Cannot launch host lambda when CUDA lambda is enabled.
-
+#if defined(KOKKOS_ENABLE_CUDA) && defined(KOKKOS_COMPILER_NVCC)
```

Comment:
Let's at least provide a comment explaining that the use case is higher (but constant) and that we see the expected numbers if we were to turn the lambda into a functor.

Reply:
Please take care of it I am swamped

Reply:
fb12f3f